runtime/src/qio/qbuffer.c

```diff
@@ -148,14 +148,8 @@ qioerr _qbytes_init_iobuf(qbytes_t* ret)
 {
   void* data = NULL;
   
-  // allocate 4K-aligned (or page size aligned)
-  // multiple of 4K
-  data = qio_valloc(qbytes_iobuf_size);
+  data = qio_memalign(sys_page_size(), qbytes_iobuf_size);
```

Comment:
Note that in some use cases, these channel buffers are just the result of running 'mmap' so they won't necessarily be in the Chapel heap either way.